src/main/scala/mesosphere/chaos/http/impl/HttpServiceImpl.scala

```diff
@@ -0,0 +1,74 @@
+package mesosphere.chaos.http.impl
+
+import java.util
+import javax.servlet.{Filter, Servlet, DispatcherType}
+
+import com.sun.jersey.api.core.DefaultResourceConfig
+import com.sun.jersey.spi.container.servlet.ServletContainer
+import org.eclipse.jetty.servlet.{FilterHolder, ServletHolder}
+import org.slf4j.LoggerFactory
+import mesosphere.chaos.http.HttpService
+
+import scala.util.Try
+/**
+  * Wrapper for starting and stopping the HttpServer.
+  */
+class HttpServiceImpl(httpServer: HttpServer) extends HttpService {
+  val servletPath = "/*"
+
+  private[this] val log = LoggerFactory.getLogger(getClass.getName)
+  private[this] var started = false
+
+  def startUp() {
+    log.debug("Starting up HttpServer.")
+    try {
+      started = true
+      preStartUp()
```

Comment:
Should preStartUp only be called once? 
Should be `started` renamed to `initialized`?
So we can write:
```scala
if (!initialized) preStartUp()
initialized = true
```
Perhaps also rename `preStartUp`